modules/core/src/webgl/classes/query.js

```diff
@@ -2,19 +2,12 @@
 import Resource from './resource';
 import {FEATURES, hasFeatures} from '../features';
 import {isWebGL2} from '../utils';
-import queryManager from '../utils/query-manager';
 import {assert} from '../../utils';
 
-const ERR_GPU_DISJOINT = 'Disjoint GPU operation invalidated timer queries';
-const ERR_TIMER_QUERY_NOT_SUPPORTED = 'Timer queries require "EXT_disjoint_timer_query" extension';
-
-const GL_QUERY_COUNTER_BITS_EXT = 0x8864; // # bits in query result for the given target.
-
 const GL_QUERY_RESULT = 0x8866; // Returns a GLuint containing the query result.
```

Comment:
Unrelated: We should import `GL.` constants instead of defining our own, since we now have a babel plugin that inlines them. 

These local defs were originally added to save bundle size as the imported refs did not minimize well, but serve no purpose now.